File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-location_item.ts


import { ISaveTrainingEventLocation_Item } from './isave-training-event-location_item';

export class SaveTrainingEventLocation_Item implements ISaveTrainingEventLocation_Item {
  
	public TrainingEventLocationID?: number;
	public LocationID: number = 0;
	public EventStartDate: Date = new Date(0);
	public EventEndDate: Date = new Date(0);
	public TravelStartDate?: Date;
	public TravelEndDate?: Date;
	public ModifiedByAppUserID?: number;
	public ModifiedDate?: Date;
	public LocationName: string = "";
	public AddressLine1: string = "";
	public AddressLine2: string = "";
	public AddressLine3: string = "";
	public CityID: number = 0;
	public CityName: string = "";
	public StateID: number = 0;
	public StateName: string = "";
	public StateCode: string = "";
	public CountryID: number = 0;
	public CountryName: string = "";
	public CountryCode: string = "";
  
}

File: GTTS/INL.TrainingService.Models/SaveTrainingEventLocation_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventLocation_Item : ISaveTrainingEventLocation_Item
    {
        public int? TrainingEventLocationID { get; set; }
        public long LocationID { get; set; }
        public DateTime EventStartDate { get; set; }
        public DateTime EventEndDate { get; set; }
		public DateTime? TravelStartDate { get; set; }
		public DateTime? TravelEndDate { get; set; }
		public int? ModifiedByAppUserID { get; set; }
        public DateTime? ModifiedDate { get; set; }

        public string LocationName { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string AddressLine3 { get; set; }
        public int CityID { get; set; }
        public string CityName { get; set; }
        public int StateID { get; set; }
        public string StateName { get; set; }
        public string StateCode { get; set; }
        public int CountryID { get; set; }
        public string CountryName { get; set; }
        public string CountryCode { get; set; }
    }
}
